nodebuilder/state/cmds/state.go

```diff
@@ -30,24 +35,27 @@ func init() {
 	)
 }
 
-var stateCmd = &cobra.Command{
+var Cmd = &cobra.Command{
 	Use:   "state [command]",
 	Short: "Allows interaction with the State Module via JSON-RPC",
 	Args:  cobra.NoArgs,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
how does this operate with the `PersistentPreRunE` func inside of `util.go` ?

Reply:
It does not. `util.PersistentPreRunE` operates only with full/light/bridge node cmds.